app/pb_kits/playbook/pb_layout/body.rb

```diff
@@ -2,38 +2,18 @@
 
 module Playbook
   module PbLayout
-    class Body < Playbook::PbKit::Base
-      PROPS = %i[configured_classname
-                 configured_data
-                 configured_id
-                 block].freeze
+    class Body
+      include Playbook::Props
 
-      def initialize(classname: default_configuration,
-                     data: default_configuration,
-                     id: default_configuration,
-                     &block)
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.block = block_given? ? block : nil
-      end
-
-      def yield(context:)
-        context.capture(&block)
-      end
+      partial "pb_layout/body"
 
-      def to_partial_path
-        "pb_layout/body"
+      def classname
+        generate_classname("layout_body")
       end
 
-    private
-
-      DEFAULT = Object.new
-      private_constant :DEFAULT
-      def default_configuration
-        DEFAULT
+      def yield(context:)
+        context.capture(&children)
```

Comment:
We've removed the custom `yield` methods in the new format. It's better to just `<%= capture(&object.children) %>` inside the template itself, instead of passing the template into the view object to invoke a method on it.